src/app/stacktrace.h

```diff
@@ -38,7 +38,7 @@ static inline void print_stacktrace(FILE *out = stderr, unsigned int max_frames
     int functionNamesFound = 0;
     // iterate over the returned symbol lines. skip the first, it is the
     // address of this function.
-    for (int i = 2; i < addrlen; i++) {
+    for (int i = 2; i < addrlen; ++i) {
```

Comment:
Aren't stacktrace 3rd party?

Reply:
maybe? but we have multiple edits in history.

Reply:
I think we should either integrate it into our codebase or don't touch at all.

Reply:
Although I would consider boost stacktrace as a replacement.